Module/Rule.Registry/Convert/Functions.SingleLine.ps1

```diff
@@ -224,60 +257,66 @@ function Get-RegistryValueTypeFromSLStig
 
     $valueName = Get-RegistryValueNameFromSingleLineStig -CheckContent $CheckContent
 
-    foreach ($key in $Hashtable.Keys)
+    if($CheckContent -match 'Wow6432Node\\McAfee')
```

Comment:
Suggestion: For this, I think it would make sense to put a comment before this line, explaining why we're forcing anything that's detected as "Wow6432Node\McAfee" in CheckContent as a "DWord".  Something like, "McAfee STIG isn't written in a way that ValueType can be detected via CheckContent and/or FixText."